src/index.ts

```diff
@@ -317,6 +317,21 @@ export default class TRAPIQueryHandler {
     this.finalizedResults = fixedResults;
   }
 
+  appendOriginalCuriesToResults(results: TrapiResult[]): void {
+    results.forEach(result => {
+      Object.entries(result.node_bindings).map(([qNodeID, bindings]) => {
```

Comment:
This could be a `forEach` since it accomplishes its job by mutating the node bindings by reference, and the return value of the map goes unused.

Reply:
Done! I was re-using similar code, missed this.